src/api/app/controllers/source_controller.rb

```diff
@@ -56,15 +56,10 @@ def global_command
 
   def projectlist
     # list all projects (visible to user)
-    output = Rails.cache.fetch(['projectlist', Project.maximum(:updated_at), User.current.forbidden_project_ids]) do
```

Comment:
did you measure this? there is a reason for this custom code - this route was one of the hot paths in new relic.
susestudio's auto complete hammers on this route while you type project names. and you're not caching the rendering.